public/components/panel/sampleSynthesizerPanel/hardwareSocketHandler.js

```diff
@@ -0,0 +1,92 @@
+import io from 'socket.io-client'
+
+// A wait time is required since hardware must be up and running
+// before a socket connection is attempted
+const delayMilliseconds = 1500
+const socketConnectionURL = 'http://0.0.0.0:5333'
```

Comment:
Should we have this as an ENV variable so we dont have to change the .js and .py code if this port changes?

Reply:
Great idea